astropy/cosmology/parameters.py

```diff
@@ -63,7 +63,8 @@
 # LOCAL
 from .realizations import available
 
-__all__ = ["available"] + list(available)
+__all__ = ["available"]
+__all__ += list(available)
```

Comment:
```suggestion
__all__ = ["available", *list(available)]
```

Why not this?

Reply:
This change comes from my understanding of https://github.com/python/typing/blob/master/docs/source/libraries.rst#library-interface-public-and-private-symbols.
typing and static type checkers don't understand "computed" ``__all__`` except for a select few operations.
This change is compliant with RUF005 and forward-looking to typing and type-using IDE tools like VSCode.